recipes/pyspark-flame/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyspark-flame" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 807099c77fbaefaeb71217995bd22d203c697c6ce32c8e75ff53e1b1b56d9322
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pypandoc
+  host:
+    - pip
+    - pyspark
```

Comment:
I think this package needs ``pyspark`` just to run the project, right?
```suggestion
```

Reply:
I'd have thought so yes, this came about as part of running the `conda skeleton` CLI.

Reply:
``conda-build skeleton`` is not the best tool to generate clean recipes for ``conda``. [Grayskull](https://github.com/marcelotrevisani/grayskull) was started to generate better recipes for conda and eventually replace ``conda-build skeleton``

https://conda-forge.org/blog//2020/03/05/grayskull

Reply:
Done.

Reply:
When I remove this, conda build no longer respects that pyspark requires python <3.8 and so the CI fails.

Reply:
Sorry I misspoke, this is a result of adding `noarch: python`.